Add Sample Documentation (#55)
LM-Development/aks-sample: Samples/PublicSamples/RecordingBot/docs/tutorials/deploy/acr.md

# Deploy an Azure Container Registry

After we successfully [deployed an AKS cluster](./aks.md), we also need a container registry. In the container registry we will later build and store the docker images of the recording bot application.  

> From the registry the AKS cluster will pull the docker images and start them on the nodes of the AKS cluster.

Reviewer: ```suggestion
The AKS cluster will pull the docker images from the registry, distributes and starts them on the nodes of the AKS cluster.

```

  "powerState": {
    "code": "Running"
  },
  "privateFqdn": null,
  "privateLinkResources": null,
  "provisioningState": "Succeeded",
  "publicNetworkAccess": null,
  "resourceGroup": "recordingbottutorial",
  "resourceUid": "0123456789abcdef12345678",
  "securityProfile": {
    "azureKeyVaultKms": null,
    "defender": null,
    "imageCleaner": null,
    "workloadIdentity": null
  },
  "serviceMeshProfile": null,
  "servicePrincipalProfile": {
    "clientId": "msi",
    "secret": null
  },
  "sku": {
    "name": "Base",
    "tier": "Free"
  },
  "storageProfile": {
    "blobCsiDriver": null,
    "diskCsiDriver": {
      "enabled": true
    },
    "fileCsiDriver": {
      "enabled": true
    },
    "snapshotController": {
      "enabled": true
    }
  },
  "supportPlan": "KubernetesOfficial",
  "systemData": null,
  "tags": null,
  "type": "Microsoft.ContainerService/ManagedClusters",
  "upgradeSettings": null,
  "windowsProfile": {
    "adminPassword": null,
    "adminUsername": "azureuser",
    "enableCsiProxy": true,
    "gmsaProfile": null,
    "licenseType": null
  },
  "workloadAutoScalerProfile": {
    "keda": null,
    "verticalPodAutoscaler": null
  }
}
```

> So now we successfully attached our Azure container registry to our AKS cluster and our AKS cluster can pull docker images from our Azure container registry.

Reviewer: ```suggestion
We successfully attached our Azure container registry to our AKS cluster and it can pull docker images from our Azure container registry.

```
